src/components/common/MovieCard.tsx

```diff
@@ -1,22 +1,31 @@
 import React from "react";
 import Movie from "../../models/Movie";
 import { MovieCardProps } from "../../interfaces/MovieCard";
+import { useParams } from "react-router";
+import { Link } from "react-router-dom";
 
 const MovieCard: React.FC<MovieCardProps> = ({ movie }: MovieCardProps) => {
+
+
+ 
+
   return (
     <div className="flex">
       <div className="m-2">
         <div className="w-72 shadow-[2px_0px_100px_0px_rgba(0,0,0,0.20)] mx-2 flex flex-col text-center justify-center items-center">
-          <h6 className="flex justify-center text-sm font-bold overflow-hidden">{movie.title}</h6>
+          <h6 className="flex justify-center text-sm font-bold overflow-hidden">
+            {movie.title}
+          </h6>
           <div className="flex flex-col">
             <img
               className="w-32 h-40"
               src={`https://image.tmdb.org/t/p/w500/${movie.poster_path}`}
               alt=""
             />
-            <h5>
-              {movie.release_date.substring(0, 4)}
-            </h5>
+            <h5>{movie.release_date.substring(0, 4)}</h5>
```

Comment:
nem rossz megoldás, de szerintem elegánsabb lenne, ha a date osztályt használnák erre.
`new Date(movie.release_date.substring).getFullYear()`
Már csak azért is mert a te megoldásod  csak az első 4 karaktert adja vissza a stringből, és ez csak akkor működik, ha a dátum formátuma mindig azonos.

 Amit én írtam az általánosabban használható, mivel a getFullYear() metódus mindig az évszámot adja vissza, függetlenül a dátum formátumától.
 
 Persze gondolhatjuk, hogy az api ugyanazt a formátumot adja vissza mindig, de mivan ha nem?